pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -134,6 +136,50 @@ message PeerInfo {
 }
 ```
 
+### Signature Policy
```

Comment:
I am not sure that this should be in the gossipsub spec -- it is a generic pubsub policy that applies to all routing algorithms.

Reply:
Agreed, will move it.

Reply:
Was looking for a place to version it, but since it's not breaking anything necessarily, and applies to pubsub, moving it to the pubsub doc sounds good.